src/main/java/edu/cornell/kfs/fp/batch/ApproveDvsSpawnedByRecurringDvStep.java

```diff
@@ -0,0 +1,26 @@
+package edu.cornell.kfs.fp.batch;
+
+import java.util.Date;
+
+import org.kuali.kfs.sys.batch.AbstractStep;
+
+import edu.cornell.kfs.fp.service.RecurringDisbursementVoucherDocumentService;
+
+public class ApproveDvsSpawnedByRecurringDvStep extends AbstractStep {
```

Comment:
Jay: 
Your call. 
What do you want the method named? 
Do you foresee another "autoApprove" step being needed in batch where we would have to distinguish between the two?